featuretools_tsfresh_primitives/tests/test_primitives.py

```diff
@@ -1,153 +1,67 @@
-    feature_matrix, features = ft.dfs(entityset=entityset, target_entity='sessions', agg_primitives=agg_primitives)
-    assert not feature_matrix.empty
-    used_primitives = set({})
-
-    for feature in features:
-        prims = {type(base_feat.primitive) for base_feat in feature.base_features}
-        prims.add(type(feature.primitive))
-        used_primitives = used_primitives | prims
-
-    for primitive in PRIMITIVES.values():
-        assert primitive in used_primitives
-
-
-@fixture(scope='session')
-def df(entityset):
-    df = pd.merge(entityset['sessions'].df, entityset['transactions'].df, on='session_id')
-    return df[['session_id', 'transaction_time', 'amount']]
-
-
-def _comprehensive_fc_prims():
-    """Yield a tuple (fc_setting, primitive, id)"""
-    supported = {primitive.name for primitive in PRIMITIVES.values()}
-    parameters = {key: value for key, value in PARAMETERS.items() if key in supported}
-
-    # lag 0 on its own doesn't make sense
-    parameters['partial_autocorrelation'] = [x for x in parameters['partial_autocorrelation'] if
-                                             x['lag'] != 0]
-
-    for fc_name, params_list in parameters.items():
-        primitives = featuretools_tsfresh_primitives.primitives_from_fc_settings({fc_name: params_list})
-        if not isinstance(params_list, list):
-            params_list = [params_list]
-        for params, primitive in zip(params_list, primitives):
-            fc_setting = {fc_name: [params] if params else None}
-            yield (fc_setting, primitive, str(fc_setting))
-
-
-@pytest.mark.parametrize('fc_setting,primitive',
-                         [(x[0], x[1]) for x in _comprehensive_fc_prims()],
-                         ids=[x[2] for x in _comprehensive_fc_prims()])
-def test_primitive(entityset, df, fc_setting, primitive):
-    expected = extract_features(df,
-                                column_id='session_id',
-                                column_sort='transaction_time',
-                                default_fc_parameters=fc_setting)
-    actual, _ = ft.dfs(entityset=entityset,
-                       max_depth=1,
-                       target_entity='sessions',
-                       trans_primitives=[],
-                       where_primitives=[],
-                       agg_primitives=[primitive])
-    actual = actual.filter(regex='transactions.amount')
-
-    numpy.testing.assert_almost_equal(expected.values, actual.values)
+def parametrize():
+    values = {'argvalues': [], 'ids': []}
+    fc_parameters = comprehensive_fc_parameters()
+    es = ft.demo.load_mock_customer(return_entityset=True)
+
+    for primitive in supported_primitives():
+        parameters = fc_parameters[primitive.name] or [{}]
+        primitives = {primitive.name: parameters}
+        primitives = primitives_from_fc_settings(primitives)
+        items = zip(parameters, primitives)
+
+        for parameters, primitive in items:
```

Comment:
Can we use a different variable than `parameters` here, to differentiate from the `parameters` above?  One is referring to a list of parameter settings and the other is a particular parameter setting, I think.